backend/tournesol/views/stats.py

```diff
@@ -104,13 +113,19 @@ def get(self, request):
 
             comparisons = Comparison.objects.filter(poll=poll)
             comparison_count = comparisons.count()
+            if len(comparisons) > 0:
+                print("Comparison", comparisons[0])
             last_month_comparison_count = comparisons.filter(
                 datetime_lastedit__gte=time_ago(days=self._days_delta)
             ).count()
+            last_week_comparison_count = comparisons.filter(
+                datetime_lastedit__gte=time_ago(days=self._days_delta_week)
```

Comment:
This would be a bit incorrect, because `timeago` used like this would return the datetime Monday at some hours:minutes depending on when we call it. I think we want instead to fix the datetime to the Monday at 00:00 of the current week

Reply:
I will look into this, because since I was just passing the `number` of Days as the parameter for filter. I would need to create a more robust filter and also look into the `comparisons` array to see how to get more specific to get Monday 00:00